handlers/bootstrap.go

```diff
@@ -130,6 +130,17 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 
 	volConfig := map[string]struct{}{"/var/run/docker.sock": {}}
 	envVars := []string{"CATTLE_PHYSICAL_HOST_UUID=" + machine.ExternalId}
+	for key, value := range machine.Labels {
+		hostLabel := ""
+		switch value.(type) {
```

Comment:
It's not strongly typed.It is an interface type. We cannot configure it in cattle as map[string]string. The best we can get is map[string]. This will always be a string, so we dont have to worry.